### PR TITLE
CacheList Splitting Configuration Modification

### PR DESCRIPTION
**Pull request by jonoommen**
CacheList Splitting Configuration Modification. Adding arguments
to verbose logging and remove code duplication.

Signed-off-by: Jonathan Oommen <jon.oommen@gmail.com>

### CODE REVIEW

**Comment by jonoommen** [comment]
This PR depends on eclipse/omr#2849.

File: runtime/gc_verbose_old_events/VerboseEventGCInitialized.cpp [hunk 1]
```diff
@@ -86,6 +86,9 @@ MM_VerboseEventGCInitialized::formattedOutput(MM_VerboseOutputAgent *agent)
 	agent->formatAndOutput(static_cast<J9VMThread*>(_omrThread->_language_vmthread), indentLevel + 1, "<attribute name=\"requestedPageSize\" value=\"0x%zx\" />", _event.heapRequestedPageSize);
 	agent->formatAndOutput(static_cast<J9VMThread*>(_omrThread->_language_vmthread), indentLevel + 1, "<attribute name=\"requestedPageType\" value=\"%s\" />", _event.heapRequestedPageType);
 	agent->formatAndOutput(static_cast<J9VMThread*>(_omrThread->_language_vmthread), indentLevel + 1, "<attribute name=\"gcthreads\" value=\"%zu\" />", _event.gcThreads);
```
[the remark this answers is not in the document]
Comment (jonoommen) [reply runtime/gc_verbose_old_events/VerboseEventGCInitialized.cpp]: Done 👍

**Comment by DanHeidinga** [comment]
@amicic @dmitripivkine Is this still relevant?

**Comment by amicic** [comment]
pretty much a duplicate of https://github.com/eclipse/openj9/pull/8409

**Pull request closed without merging**